.github/workflows/tests.yml

```diff
@@ -40,7 +40,7 @@ jobs:
                   node-version: "16"
             - name: Make git use https instead of ssh
               run: git config --global url."https://github.com/".insteadOf ssh://git@github.com/
-            - run: npm i
+            - run: npm i --force
```

Comment:
this change is already in branch 9.2. Also, since this is nothing to do with this PR, it should not be here in the first place.